plugins/python-build/bin/python-build

```diff
@@ -166,6 +166,60 @@ can_use_macports() {
   PYTHON_BUILD_SKIP_MACPORTS=1; return 1
 }
 
+can_use_pixi() {
+  if locked_in; then
+    locked_in pixi && rc=$? || rc=$?; return $rc
+  fi
+  [[ -n "$PYTHON_BUILD_USE_PIXI" && -n "$PYTHON_BUILD_SKIP_PIXI" ]] && {
+    echo "error: mutually exclusive environment variables PYTHON_BUILD_USE_PIXI and PYTHON_BUILD_SKIP_PIXI are set" >&3
+    exit 1
+  }
+  [[ -n "$PYTHON_BUILD_USE_PIXI" ]] && return 0
+  [[ -n "$PYTHON_BUILD_SKIP_PIXI" ]] && return 1
+  # Return cached result if already checked
+  [[ -n "$PYTHON_BUILD_PIXI_ENVS_DIR" ]] && return 0
+  # Check if pixi exists and cache the environment directory
+  local envs_dir
+  command -v pixi &>/dev/null && \
+    envs_dir="$(pixi info 2>/dev/null | grep 'Environment dir:' | awk '{print $3}')" && \
+    [[ -n "$envs_dir" ]] && { export PYTHON_BUILD_PIXI_ENVS_DIR="$envs_dir"; return 0; }
+
+  # do not check the same stuff multiple times
+  PYTHON_BUILD_SKIP_PIXI=1; return 1
+}
+
+pixi_env_prefix() {
+  # Return cached value if available
+  if [[ -n "${PYTHON_BUILD_PIXI_PREFIX:-}" ]]; then
+    echo "$PYTHON_BUILD_PIXI_PREFIX"
+    return 0
+  fi
+
+  # Use PYTHON_BUILD_PIXI_ENV if set, otherwise default to "python-deps"
+  local env_name="${PYTHON_BUILD_PIXI_ENV:-python-deps}"
```

Comment:
Where does this name come from? Is it special? Does the user need to create this conda environment manually?

Reply:
> To be honest, I'm not sure what the recommended practice is

Well, I don't know that, either! I've learned about this tool's existence from you two days ago!

Maybe it's worth consulting its maintainers on this topic?